glean-core/csharp/Glean/Dispatchers.cs

```diff
@@ -103,16 +110,17 @@ internal static Task LaunchAPI(Action action)
                     // task asynchronously                    
                     task = factory.StartNew(() =>
                     {
-                            // In order to prevent tasks from causing exceptions
-                            // we wrap the action invocation in try/catch
-                            try
+                        // In order to prevent tasks from causing exceptions
+                        // we wrap the action invocation in try/catch
+                        try
                         {
                             action.Invoke();
                         }
                         catch (Exception)
                         {
-                                //TODO Exception eaten by Glean
-                            }
+                            Log.Error("Exception thrown by queued initial task and swallowed.");
```

Comment:
```suggestion
                            Log.Error(e, "Exception thrown by queued initial task and swallowed.");
```

... then we can use it here to print useful information :)